suite-native/module-demo-account-questionnaire/src/screens/DemoAccountQuestionnaireIntroScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import { useCallback } from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { EventType, analytics } from '@suite-native/analytics';
+import { Button, InlineAlertBox, PictogramTitleHeader, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import {
+    DemoAccountQuestionnaireStackParamList,
+    DemoAccountQuestionnaireStackRoutes,
+    Screen,
+    ScreenHeader,
+    StackProps,
+} from '@suite-native/navigation';
+
+export const DemoAccountQuestionnaireIntroScreen = () => {
+    const navigation =
+        useNavigation<
+            StackProps<
+                DemoAccountQuestionnaireStackParamList,
+                DemoAccountQuestionnaireStackRoutes.Intro
+            >['navigation']
+        >();
+
+    const handleContinue = useCallback(() => {
+        analytics.report({ type: EventType.DemoAccountQuestionnaireStart }, { force: true });
+        navigation.navigate(DemoAccountQuestionnaireStackRoutes.Reason);
+    }, [navigation]);
+
+    const handleExit = useCallback(() => {
+        if (navigation.canGoBack()) {
+            navigation.goBack();
+        }
+    }, [navigation]);
+
+    return (
+        <Screen
+            header={<ScreenHeader closeActionType="back" />}
+            footer={
+                <VStack spacing="sp12" paddingHorizontal="sp16" marginBottom="sp16">
+                    <Button onPress={handleContinue} isFullWidth>
+                        <Translation id="moduleDemoAccountQuestionnaire.intro.primaryCta" />
+                    </Button>
+                    <Button colorScheme="tertiaryElevation0" onPress={handleExit} isFullWidth>
+                        <Translation id="moduleDemoAccountQuestionnaire.intro.secondaryCta" />
+                    </Button>
+                </VStack>
+            }
+        >
+            <VStack spacing="sp32" alignItems="center" justifyContent="center" flex={1}>
+                <VStack justifyContent="center">
+                    <PictogramTitleHeader
+                        variant="success"
+                        icon="handWaving"
+                        title={<Translation id="moduleDemoAccountQuestionnaire.intro.title" />}
+                        subtitle={
+                            <Translation id="moduleDemoAccountQuestionnaire.intro.subtitle" />
+                        }
+                        titleVariant="titleMedium"
+                    />
+                </VStack>
+                <VStack spacing="sp16" alignItems="center">
```

Comment:
nit: VStack has no effect here.
```suggestion
                <Box spacing="sp16" alignItems="center">
```

Reply:
👍 Found out, the wrapper was completely redundant 6c617db58651bbdffb35c0b0a32eb5fee51feb19